### DISCUSSION TITLE
Doc UX: Algolia false email mask on "Fastly Compute@Edge"

### DISCUSSION THREAD

**Discussion by mountainash** [honojs/hono#1040] [General]
Seriously minor issue, but maybe a small fix for the Admin with Algolia access.

<img width="585" alt="Screenshot 2023-04-24 at 2 49 42 pm" src="https://user-images.githubusercontent.com/27637/234002915-e8638bee-a4d6-4da8-b311-079709cd94f5.png">

**Comment by yusukebe** [comment]
This issue occurs because the titles include '@', causing them to be detected as email addresses and it is masked. I'm contacting Algolia support to find out how to fix it.

**Comment by yusukebe** [comment]
This is fixed now!
